newIDE/app/src/AssetStore/PrivateGameTemplates/PrivateGameTemplatePurchaseDialog.js

```diff
@@ -0,0 +1,370 @@
+// @flow
```

Comment:
Would it be hard to reuse/refactor the purchase dialog for asset packs?

Reply:
I thought about it, and at the same time, I'd rather keep it different. 
The pack page displays similar packs, where this one doesn't yet, plus the description is slightly different.
In the future, I feel that they will diverge